pkg/testutils/blockchain.go

```diff
@@ -0,0 +1,11 @@
+package testutils
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+func IntToHash(x int64) common.Hash {
```

Comment:
Maybe `Int64ToHash` in case we need other functions for other types of ints?

Reply:
Makes sense, renamed in the last commit.